packages/@okta/vuepress-site/use_cases/inline_hooks/registration_hook/registration_hook/index.md

```diff
@@ -0,0 +1,244 @@
+      }
+   ]
+}
+```
+
+Registrations are allowed by default, so setting a value of `ALLOW` for the `action` field is valid but superfluous. 
+
+### error
+
+See [error](/use_cases/inline_hooks/) for general information on the structure to use for the `error` object.
+
+In the case of the Registration Inline Hook, the `error` object provides a way of causing an error message to be displayed to the end user who is trying to register. If you're using the Okta Sign-In Widget for Self-Service Registration, and have not customized its error handling behavior, what is displayed to the end user is the `errorSummary` of the first `ErrorCause` object that your external service returns.
+
+If you do not return any value for that `errorCauses` object, but deny the user's registration attempt via the `commands` object in your response to Okta, the following generic message is displayed to the end user: "Registration cannot be completed at this time".
+
+## Sample Listing of JSON Payload of Request
+
+```json
+{
+   "eventType":"com.okta.user.pre-registration",
+   "eventTypeVersion":"1.0",
+   "cloudEventVersion":"0.1",
+   "source":"/api/v1/registration/reg17jsXFYSfBEkZh0g4/register",
+   "eventID":"763F35F3-7D83-4547-836B-F55382ADBDC7",
+   "eventTime":"2018-05-29T20:04:22Z",
+   "contentType":"application/json",
+   "data":{
+      "context":{
+         "request":{
+            "id":"reqa-0DCpdKQe-td_UodXKpag",
+            "method":"POST",
+            "url":{
+               "value":"http://example.okta.com/api/v1/registration/reg17jsXFYSfBEkZh0g4/register"
+            },
+            "ipAddress":"192.168.122.1",
+            "locale":"en-US",
+            "client":""
+         }
+      },
+      "user":{
+         "profile":{
+            "firstName":"Isaac",
+            "lastName":"Brock",
+            "login":"isaac.brock@example.com",
+            "mobilePhone":"555-415-1337"
+         }
+      }
+   }
+}
+```
+
+## Sample Listing of JSON Payload of Response
+
+```json
+{
+   "commands":[
+      {
+         "type":"com.okta.action.update",
+         "value":{
+            "action":"DENY"
+         }
+      }
+   ],
+   "error":{
+      "errorSummary":"Errors were found in the user profile",
+      "errorCauses":[
+         {
+            "errorSummary":"You specified an invalid email domain",
+            "reason":"INVALID_EMAIL_DOMAIN",
+            "locationType":"body",
+            "location":"data.userProfile.login",
+            "domain":"end-user"
+         }
+      ]
+   },
+   "debugContext":{
+      "executionTimeMillis":231
+   }
+}
+```
+
+## Enabling a Registration Inline Hook for Self-Service Registration
+
+To activate the inline hook, you first need to register your external service endpoint with Okta using the [Inline Hooks Management API](/docs/api/resources/inline-hooks).
+
+You then need to associate the registered inline hook with your Self-Service Registration policy. (What to do if you've not yet configured a policy...)
```

Comment:
It seems I forgot to follow up on this: Should we include steps for configuring a simple registration policy here? Or perhaps we could link to another page that describes how to do that (though I don't see any existing documentation for SSR). Thoughts, @wdawson-okta?

Reply:
Might be good to link to the help.okta.com page for it?

Reply:
Yeah, that works. @motikagan-okta : https://help.okta.com/en/prod/Content/Topics/Directory/eu-self-service.htm

Reply:
Added link to SSR info on help.okta.com.